context/app/static/js/components/organ/Samples/Samples.jsx

```diff
@@ -60,19 +70,31 @@ function Samples({ organTerms }) {
       },
       _source: [...columns.map((column) => column.id), 'donor.mapped_metadata.age_unit'],
       size: 10000,
+      sort:
+        sortState?.columnId && columnMap[sortState.columnId].sortTerm
+          ? [{ [columnMap[sortState.columnId].sortTerm]: sortState.direction }]
+          : undefined,
     }),
-    [organTerms],
+    [organTerms, sortState],
   );
 
-  const { searchHits } = useSearchHits(query);
+  const previousSearchHits = useRef([]);
+
+  const { searchHits, isLoading } = useSearchHits(query);
```

Comment:
We could potentially introduce something like https://swr.vercel.app/docs/middleware.en-US#keep-previous-result so we could leverage this elsewhere.

Reply:
Great find! I didn't know this functionality could be encapsulated into middleware, that makes a lot of sense. I'll make this revision!

Reply:
Implemented in [`e8746552087d98fb392b5ce7c833bf4571353252..4da59e3d8eda41c512f98aca92a99fcbfd5e0660`](https://github.com/hubmapconsortium/portal-ui/pull/3232/files/e8746552087d98fb392b5ce7c833bf4571353252..4da59e3d8eda41c512f98aca92a99fcbfd5e0660)